apps/right-to-rent-check/acceptance/features/tenant-in-uk.js

```diff
@@ -35,9 +35,9 @@ Scenario('When I select Yes And submit the form Then I am taken to /current-prop
 
 Scenario('When I select No And submit the form Then I am taken to the /uk-check-yourself page', (
```

Comment:
Should this now say `/check-not-needed-uk` ?

Reply:
yes, I will amend